provisioning/provisioning-service-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/service/implementation/ProvisioningServiceClientImpl.java

```diff
@@ -0,0 +1,1823 @@
+import com.microsoft.azure.sdk.iot.provisioning.service.models.QueryEnrollmentGroupsHeaders;
+import com.microsoft.azure.sdk.iot.provisioning.service.models.QueryIndividualEnrollmentsHeaders;
+import com.microsoft.azure.sdk.iot.provisioning.service.models.QuerySpecification;
+import com.microsoft.rest.ServiceCallback;
+import com.microsoft.rest.ServiceFuture;
+import com.microsoft.rest.ServiceResponse;
+import com.microsoft.rest.ServiceResponseWithHeaders;
+import com.microsoft.rest.Validator;
+import java.io.IOException;
+import java.util.List;
+import okhttp3.ResponseBody;
+import retrofit2.http.Body;
+import retrofit2.http.GET;
+import retrofit2.http.Header;
+import retrofit2.http.Headers;
+import retrofit2.http.HTTP;
+import retrofit2.http.Path;
+import retrofit2.http.POST;
+import retrofit2.http.PUT;
+import retrofit2.http.Query;
+import retrofit2.Response;
+import rx.functions.Func1;
+import rx.Observable;
+
+/**
+ * Initializes a new instance of the ProvisioningServiceClient class.
+ */
+public class ProvisioningServiceClientImpl extends ServiceClient implements ProvisioningServiceClient {
+    /**
+     * The Retrofit service to perform REST calls.
+     */
+    private ProvisioningServiceClientService service;
+
+    /** The API version to use for the request. Supported versions include: 2018-09-01-preview. */
+    private String apiVersion;
+
+    /**
+     * Gets The API version to use for the request. Supported versions include: 2018-09-01-preview.
+     *
+     * @return the apiVersion value.
+     */
+    public String apiVersion() {
+        return this.apiVersion;
+    }
+
+    /**
+     * Sets The API version to use for the request. Supported versions include: 2018-09-01-preview.
+     *
+     * @param apiVersion the apiVersion value.
+     * @return the service client itself
+     */
+    public ProvisioningServiceClientImpl withApiVersion(String apiVersion) {
+        this.apiVersion = apiVersion;
+        return this;
+    }
+
+    /**
+     * Initializes an instance of ProvisioningServiceClient client.
+     */
+    public ProvisioningServiceClientImpl() {
+        this("https://your-dps.azure-devices-provisioning.net");
+    }
+
+    /**
+     * Initializes an instance of ProvisioningServiceClient client.
+     *
+     * @param baseUrl the base URL of the host
+     */
+    public ProvisioningServiceClientImpl(String baseUrl) {
+        super(baseUrl);
+        initialize();
+    }
+
+    /**
+     * Initializes an instance of ProvisioningServiceClient client.
+     *
+     * @param clientBuilder the builder for building an OkHttp client, bundled with user configurations
+     * @param restBuilder the builder for building an Retrofit client, bundled with user configurations
+     */
+    public ProvisioningServiceClientImpl(OkHttpClient.Builder clientBuilder, Retrofit.Builder restBuilder) {
+        this("https://your-dps.azure-devices-provisioning.net", clientBuilder, restBuilder);
```

Comment:
Here as well. What purpose does this function serve? There is no "your-dps.azure-devices-provisioning.net"

Reply:
Again, auto-generated code